bucketdownload/activity_test.go

```diff
@@ -203,7 +217,16 @@ func TestActivity(t *testing.T) {
 			assert.NilError(t, err)
 
 			var result bucketdownload.Result
-			_ = enc.Get(&result)
+			err = enc.Get(&result)
+			assert.NilError(t, err)
+
+			// Missing DirPath param, one check the created temp directory contents.
```

Comment:
I think "one" was supposed to be "only" in this comment?  It should also be rewritten as a complete sentence.